### PR TITLE
[clojure-package] fix docstrings in `normal.clj`

### PR DESCRIPTION
**Pull request by Chouffe**
## Description ##

* Added spec to catch `high` < `low` in `uniform`
* Added spec to catch `scale` <= 0 in `normal`
* Added unit tests

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x ] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by gigasquid** [comment]
Thanks for your contribution @Chouffe. I look forward to reviewing it shortly 😸

File: contrib/clojure-package/src/org/apache/clojure_mxnet/random.clj [hunk 1]
```diff
@@ -16,70 +16,84 @@
...
-    returns: The result ndarray with generated result./"
+  "Generate uniform distribution in [`low`, `high`) with shape.
+    `low`: The lower bound of distribution.
+    `high`: The upper bound of distribution.
+    `shape-vec`: vector shape of the ndarray generated.
+    `opts-map` {
+      `ctx`: Context of output ndarray, will use default context if not specified.
+      `out`: Output place holder}
+    returns: The result ndarray with generated result.
+   Ex:
```
Comment (gigasquid) [inline contrib/clojure-package/src/org/apache/clojure_mxnet/random.clj]: Examples in the docstring are very nice 💯

File: contrib/clojure-package/src/org/apache/clojure_mxnet/random.clj [hunk 2]
```diff
@@ -16,70 +16,84 @@
...
+   as well as results from executors that contains Random number
+   such as Dropout operators.
 
-   seed-state: The random number seed to set to all devices.
+   `seed-state`: The random number seed to set to all devices.
    note: The random number generator of mxnet is by default device specific.
          This means if you set the same seed, the random number sequence
-         generated from GPU0 can be different from CPU."
+         generated from GPU0 can be different from CPU.
+   Ex:
```
Comment (gigasquid) [inline contrib/clojure-package/src/org/apache/clojure_mxnet/random.clj]: 😍

**Comment by gigasquid** [comment]
It is good to go once the CI tests pass

**Comment by Chouffe** [comment]
The main issue was in the docstring for `normal` + some extra specs for `uniform`
```
`loc`: Mean (centre) of the distribution.
`scale`: Standard deviation (spread or width) of the distribution.
```

**Comment by Chouffe** [comment]
Not sure why CI has not run?

**Comment by anirudhacharya** [comment]
@mxnet-label-bot add [pr-awaiting-merge, Clojure]

**Comment by gigasquid** [comment]
@Chouffe would you mind pushing an empty commit or something to retrigger the CI? I think it might be fixed now

**Comment by Chouffe** [comment]
I just rebased on upstream master and pushed again :)

**Comment by Chouffe** [comment]
It seems to be green @gigasquid :-)

**Comment by gigasquid** [comment]
Yay! thanks

**Pull request merged**